common/log/helpers/helpers.go

```diff
@@ -3,7 +3,7 @@ package helpers
 import (
 	log "github.com/sirupsen/logrus"
 
-	"github.com/twitter/scoot/scootapi/gen-go/scoot"
+	"github.com/twitter/scoot/scheduler/api/thrift/gen-go/scoot"
```

Comment:
having something in common use (import) from a specific service is a red flag (tends to lead to cyclic dependencies)... is LogRunStatus() used by > 1 service?  If so, then scoot.JobStatus is used by > 1 service and should also be in common.  If LogRunStatus() is only used by scheduler, then it can be moved to scheduler.

Reply:
It's used by both the scootcl thrift client and the bzutil grpc client to get the job's status - agreed though re: service imports into common